### PR TITLE
Updated SBL security related items (#73)

### PR DESCRIPTION
**Pull request by sagraw2**
1) Added a picture depicting SBL Verified Boot chain.
2) Added a table describing the usage of various keys.

Signed-off-by: Sachin Agrawal <sachin.agrawal@intel.com>

### CODE REVIEW

**Comment by lsubash** [comment]
In the verified boot flow picture while doing payload load, OS loader and FwUpdatePayload is hash verified where as UEFI  payload is container format with RSA verification.  Should we need to mention it?

File: source/security/key-management.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+|                 |                  | Policy Manifest.       |                        |
++-----------------+------------------+------------------------+------------------------+
+| ConfigData Key  | Firmware Owner   | To sign Config Data    | Use CfgDataTool.py     |
+|                 |                  | blob.                  |                        |
++-----------------+------------------+------------------------+------------------------+
+| Master Key      | Firmware Owner   | To sign |SPN|          |                        |
+|                 |                  | Key Manifest.          |                        |
++-----------------+------------------+------------------------+------------------------+
+| Container Def   | Firmware Owner   | To sign container      | Use GenContainer.py    |
+| Key             |                  | components, such as,   | to package various     |
```
Comment (mauricema) [inline source/security/key-management.rst:22]: To sign container header.

Comment (sagraw2) [reply source/security/key-management.rst]: Thanks for catching this. Updated.

**Comment by sagraw2** [comment]
> In the verified boot flow picture while doing payload load, OS loader and FwUpdatePayload is hash verified where as UEFI payload is container format with RSA verification. Should we need to mention it?

In the 'Verified Boot' picture, we have 8-y and 8-x pointing to hashes available in hash store, whereas 8-z showing UEFI Pld Container being verified using Container Def Pub Key hash.

**Pull request merged**